front/app/modules/commercial/content_builder/admin/components/ContentBuilderLayout/index.tsx

```diff
@@ -27,7 +62,22 @@ const ContentBuilderLayout: React.FC<ContentBuilderLayoutProps> = ({
     return null;
   }
 
-  return <>{children}</>;
+  return (
```

Comment:
All of this was moved to the index file in the module, I believe. Can we keep it there?

Reply:
For sure - looks like I made a mistake when merging. Fixed now!